com.unity.cinemachine/Editor/Samples/ExposeHDRPInternals/AssemblyInfo.cs

```diff
@@ -0,0 +1,2 @@
+using System.Runtime.CompilerServices;
+[assembly: InternalsVisibleTo("Unity.Cinemachine.Editor")]
```

Comment:
This feels a little dangerous.  You're exposing all of HDRP's internals to Cinemachine, making it possible for us to inadvertently depend on them in the future.  I think a better approach would be to use the asmref to *add* a single public API to HDRP, which calls the required internal method.

Reply:
That's a great idea! It's my first time with asmref, so I was happy I could make it work :)